js/src/personal/kazi/Kazi.page.tsx

```diff
@@ -1,18 +1,29 @@
-import { Title, Text } from '@mantine/core'
+import { Title, Text, useMantineColorScheme } from '@mantine/core'
 import React, { useState } from 'react'
 import { FaSun, FaMoon } from 'react-icons/fa'
 import styles from './Kazi.module.css'
 
 export function KaziPage() {
   const [sun, setSun] = useState(true)
-  const [sunAnimate, setSunAnimate] = useState(false)
+  const [animate, setAnimate] = useState('')
+  const { setColorScheme } = useMantineColorScheme()
 
-  const sunToMoon = () => {
-    setSunAnimate(true)
-    setTimeout(() => {
-      setSun(!sun)
-      setSunAnimate(false)
-    }, 2000)
+  const changeIcon = () => {
```

Comment:
With the logic it has, likely the DarkModeToggle should have its own component in its own file. 
The logic here doesn't really make sense in belonging to KaziPage
DarkModeToggle.tsx?

Reply:
Just committed DarkModeToggle.tsx component.